tests/profile/serializers/test_indicator_data_serializer.py

```diff
@@ -117,6 +161,22 @@ def test(self, profile, geography, profile_indicator, category, subcategory):
         serializer = IndicatorDataSerializer(profile, geography)
         assert serializer[category.name]["subcategories"][subcategory.name]["indicators"][""]["id"] == profile_indicator.id
 
+@pytest.mark.django_db
+@pytest.mark.usefixtures("groups")
+class TestQualitativeData:
+    def test_without_qualitative_data(self, profile, geography, profile_indicator, category, subcategory):
+        serializer = IndicatorDataSerializer(profile, geography)
+        assert serializer[category.name]["subcategories"][subcategory.name]["indicators"][""]["content"] == {}
```

Comment:
what is the empty string `[""]` after `["indicators"]` is that the name of the `profile_indicator`? If so, replace it with `[profile_indicator.name]` (or whatever works) so that it is more self-documenting